packages/image-resizer/index.js

```diff
@@ -0,0 +1,88 @@
+import { config } from './configs.js'
+import { log } from './utils.js'
+import { Storage } from '@google-cloud/storage'
+import sharp from 'sharp'
+import { join, basename, extname } from 'path'
+import { tmpdir } from 'os'
+import { promises as fs } from 'fs'
+import express from 'express'
+import bodyParser from 'body-parser'
+
+const storage =
+  config.gcs.projectId && config.gcs.keyFilename
+    ? new Storage({
+        projectId: config.gcs.projectId,
+        keyFilename: config.gcs.keyFilename,
+      })
+    : new Storage()
+
+const app = express()
+
+app.use(bodyParser.json())
```

Comment:
nitpick:
`app.use(bodyParser.json())` -> `app.use(express.json())`
`express` has `json` method, which is identical to `bodyParser.json`.